plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/BaseClickHouseConnectorTest.java

```diff
@@ -104,6 +104,18 @@ public void testAddColumnWithCommentSpecialCharacter(String comment)
         }
     }
 
+    @Override
+    public void testDropAndAddColumnWithSameName()
+    {
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_drop_add_column", "(x int NOT NULL, y int, z int) WITH (engine = 'MergeTree', order_by = ARRAY['x'])", ImmutableList.of("1,2,3"))) {
```

Comment:
Not any change..Just for my understanding - Why can't we use CTAS is it bcoz we need to have a column `NOT NULL` ?

Reply:
Right. The test uses `MergeTree` engine because  the default `Log` engine doesn't support dropping a column and it also requires `order_by` on `NOT NULL` columns.